diff --git a/Schemas/LEMS/LEMS_v0.7.6.xsd b/Schemas/LEMS/LEMS_v0.7.6.xsd
index 3843c9c..aa6c119 100644
--- a/Schemas/LEMS/LEMS_v0.7.6.xsd
+++ b/Schemas/LEMS/LEMS_v0.7.6.xsd
@@ -27,12 +27,11 @@
     elementFormDefault="qualified"
     attributeFormDefault="unqualified">
 
-    <xs:element name="Lems">
+    <xs:complexType name="Lems">
         <xs:annotation>
             <xs:documentation>The root Lems element.</xs:documentation>
         </xs:annotation>
 
-        <xs:complexType>
 
             <xs:sequence>
                 <xs:element name="Target" type="Target" minOccurs="0" maxOccurs="1"/>
@@ -46,14 +45,13 @@
 
             <xs:attribute name="description" type="xs:string" use="optional"/>
 
-        </xs:complexType>
 
         <!--<xs:unique name="dimension_uniqueness">
     		<xs:selector xpath="Dimension"></xs:selector>
     		<xs:field xpath="@name"></xs:field>
         </xs:unique>-->
 
-    </xs:element>
+    </xs:complexType>
 
 
 
