diff --git i/Schemas/NeuroML2/NeuroML_v2.3.1.xsd w/Schemas/NeuroML2/NeuroML_v2.3.1.xsd
index 741f1e8..e2efb78 100644
--- i/Schemas/NeuroML2/NeuroML_v2.3.1.xsd
+++ w/Schemas/NeuroML2/NeuroML_v2.3.1.xsd
@@ -2487,14 +2487,6 @@
       </xs:extension>
     </xs:complexContent>
   </xs:complexType>
-  <!-- TODO: remove -->
-  <xs:complexType name="ConcentrationModel_D">
-    <xs:complexContent>
-      <xs:extension base="DecayingPoolConcentrationModel">
-        <xs:attribute name="type" use="required" fixed="decayingPoolConcentrationModel"/>
-      </xs:extension>
-    </xs:complexContent>
-  </xs:complexType>
   <xs:complexType name="IntracellularProperties">
     <xs:annotation>
       <xs:documentation>Biophysical properties related to the intracellular space within the  **cell** , such as the  **resistivity**  and the list of ionic  **species**  present. **caConc**  and **caConcExt**  are explicitly exposed here to facilitate accessing these values from other Components, even though **caConcExt**  is clearly not an intracellular property
